Sources/Build/BuildPlan.swift

```diff
@@ -577,14 +577,16 @@ public final class SwiftTargetBuildDescription {
     private func generateResourceAccessor() throws {
         // Do nothing if we're not generating a bundle.
         guard let bundlePath = self.bundlePath else { return }
-
+      
+        let pathToModule = isTestTarget ? bundlePath : "Bundle.main.bundlePath/\(bundlePath.basename)"
```

Comment:
I had just realized this is not actually going to word, I've made some changes locally and building a new toolchain before pushing, but tentatively this has been re-written to 
```swift
let pathToModule = isTestTarget ? bundlePath : "Bundle.main.bundlePath + \"/\" + \(bundlePath.basename)"
```